### PR TITLE
Extending identifier mapping for Sinks

### LINKED ISSUE #29: Identifier Persistence for Sinks
As I recall this being something we mentioned in out calls but we currently have no documentation in our spec for if or when this should change.

- Can the GUID be persistent for the physical connection aka Sink and solely use the `version` (or the [capabilities version](https://specs.amwa.tv/bcp-004-01/releases/v1.0.0/docs/1.0._Receiver_Capabilities.html#capabilities-version)) change in the receiver core to signal a different physical device?
- Do we need to extent the [data model](https://specs.amwa.tv/is-04/releases/v1.3.1/docs/5.1._Data_Model_-_Identifier_Mapping.html#identifier-persistence--generation)? For when and when not to change the ID of a Sink

### PR DESCRIPTION
**Pull request by prince-chrismc**
resolves #29 

I did my best to edit the original diagram

### CODE REVIEW

File: docs/2.0 Data Model.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+![Extended IS-04 Model Diagram](../docs/images/is-11-data-model.png)
+
+The IS-04 heirarchy is preserved with the exception of the addition of a Receiver can identify an association with 0 or more Sink resources.
+
+## Sink Representation
+
+A new Sink is created in the following circumstance:
+
+- At the presence of a phycial display
```
Comment (cristian-recoseanu) [inline docs/2.0 Data Model.md]: "Physical display"

Comment (prince-chrismc) [reply docs/2.0 Data Model.md]:
```suggestion
- At the presence of a physical display
```

File: docs/2.0 Data Model.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Owned by:
+
+- Device ID
+
+SHOULD change if:
+
+- Device ID changes
+- A different physical interface (such as an HDMI input) is mapped to consume the essence
+- A different physical display is connect to the same interface (using the manufacure information. serial number, or unique identifier)
```
Comment (N-Nagorny) [inline docs/2.0 Data Model.md]:
```suggestion
- A different physical display is connect to the same interface (using the manufacturer information, serial number, or unique identifier)
```

**Review by N-Nagorny — APPROVED** [review]
Looks good to me.

**Pull request merged**